web/templates/typeahead_list_item.hbs

```diff
@@ -34,13 +34,15 @@
     {{~#if has_status}}
     {{> status_emoji status_emoji_info}}
     {{~/if}}
-    {{~#if has_secondary}}
+    {{#if has_secondary_html}}
+    <small class="autocomplete_secondary rendered_markdown">{{{secondary_html}}}</small>
+    {{else if has_secondary}}
     <small class="autocomplete_secondary">
         {{~ secondary ~}}
     </small>
+    {{/if}}
     {{#if is_unsubscribed}}
     <span class="fa fa-exclamation-triangle unsubscribed_icon"
       title="{{t 'You are not currently subscribed to this channel.' }}"></span>
     {{/if}}
-    {{~/if}}
```

Comment:
If you use `{{~/if}}` instead, I think that might avoid the whitespace changes below. You're effectively reverting 024362278ecb46d1aa1c1b7e5a3dc4abfafbc801. Did you now know what that `~` syntax does? Take some time to learn about "handlebars whitespace control" and then rework this section with that knowledge in hand.

Reply:
I did that but the newline character is also removed along with the whitespaces.

Reply:
Yeah, that's fine, we don't need it.